spec/models/concerns/user_scopes_spec.rb

```diff
@@ -61,7 +61,20 @@
     it { is_expected.to eq([sony_user]) }
   end
 
+  describe "nintendo_users" do
+    let!(:nintendo_user) { create(:user, nintendo: 'example') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.